src/sso/interfaces/authorization-url-options.interface.ts

```diff
@@ -1,6 +1,7 @@
 export interface AuthorizationURLOptions {
   domain?: string;
-  projectID: string;
+  projectID?: string | undefined;
```

Comment:
We don't actually need `| undefined` here, as making a property as optional (`?`) already widens the type to include `undefined`.

Reply:
done